src/plotter_disk.hpp

```diff
@@ -153,9 +156,14 @@ class DiskPlotter {
         }
 #endif /* defined(_WIN32) || defined(__x86_64__) */
 
+        const char is_parallel_writing_enabled = !DiskUtil::ShouldLock(final_dirname);
```

Comment:
the name suggests that this should be a `bool`, yet, it's a `char`. Is there a good reason for this?